src/sentry/auth/superuser.py

```diff
@@ -73,6 +83,13 @@ def __init__(self, request, allowed_ips=UNSET, org_id=UNSET, current_datetime=No
             self.org_id = org_id
         self._populate(current_datetime=current_datetime)
 
+    @staticmethod
+    def _need_validation():
+        try:
+            return settings.VALIDATE_SUPERUSER_ACCESS_CATEGORY_AND_REASON
+        except AttributeError:
```

Comment:
how can this `AttributeError` happen?

Reply:
i'll remove this, this won't happen if the setting is set